### ISSUE TITLE
Parameters choice

### ISSUE THREAD

**Issue by PiNo010** [kpchamp/SindyAutoencoders#6]
Hi,
I'm a PHD student and I'm trying to implement SindyAutoencoders from this paper in pytorch. I'm having some issues in making loss converge, and if it happens the system found by sindy is wrong. I have noticed that my implementation is very susceptible to parameters change (or very small difference in numeric differentiation) like weights of the loss function. I would like to ask to you how you have found the parameter of your network and if it good that small parameter change causes very different results.
Thank you very much
